examples/experimental/h3-grid/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "h3-grid",
+  "version": "0.0.0",
+  "license": "MIT",
+  "scripts": {
+    "start-local": "webpack-dev-server --env.local --progress --hot --open",
+    "start": "webpack-dev-server --progress --hot --open",
+    "build": "webpack -p --output-path dist"
+  },
+  "dependencies": {
+    "deck.gl": "^8.1.0-alpha.1",
```

Comment:
Don't you need `h3-js` in the dependencies?

Reply:
h3-js is a dependency of `@deck.gl/geo-layers`.